### DISCUSSION TITLE
Unable to generate Mocks with latest Cuckoo. Need Help!!

### DISCUSSION THREAD

**Discussion by ubitCloud** [Brightify/Cuckoo#502] [Q&A] [answered]
I am currently working on a sample project for unit testing using the Cuckoo mocking framework. For the past two weeks, I have been struggling to generate mock classes properly, and I am reaching out in hopes of finding a solution.

Recently, I encountered an issue after updating the Cuckoo version, and I am seeking your assistance to resolve it. Below are the details of the problem.

The script for generating mocks is saved at the root directory and executed manually in the terminal.

Initially, using Cuckoo version 1.10.4, the GeneratedMocks.swift file was successfully generated with the expected content. However, after updating to Cuckoo version 2.0.8, the GeneratedMocks.swift file is still being generated but is now empty. I am unable to identify the cause of this issue and need assistance in understanding what is going wrong with the generation of the mocks in the updated version.

For further details, you can refer to my Stack Overflow post where I have shared the script I am using and my understanding of the problem.

Additionally, you can find the project repository on GitHub.

Project Repo: https://github.com/ubitCloud/CuckooP2

Stackoverflow: https://stackoverflow.com/questions/78743899/generated-empty-mock-file-cuckoo-mocking-framework

**Comment by MatyasKriz** [comment, marked as answer]
Cuckoo 1.x uses different run script from 2.x, you need to update it. The error message is right there..

Also 2.x makes use of Cuckoofile.toml, so make sure to define your project in that. Consult README.

**Reply by ubitCloud** [reply]
@MatyasKriz  I have successfully generate the mocks thanks ☺️
